src/components/thumbnail-gallery/ThumbnailGallery.tsx

```diff
@@ -192,7 +197,11 @@ const ThumbnailGallery: React.FC<IThumbnailGalleryProps> = ({
                   })}
                 >
                   <ImageListItemBar
-                    style={{textAlign: titleAlignment}}
+                    style={{
+                      textAlign: titleAlignment,
+                      paddingLeft: borderRadius !== 0 && (titlePosition === TitlePosition.TOP || titlePosition === TitlePosition.BOTTOM) ? "calc(" + borderRadius / 2 + "%)" : 0,
+                      paddingRight: borderRadius !== 0 && (titlePosition === TitlePosition.TOP || titlePosition === TitlePosition.BOTTOM) ? "calc(" + borderRadius / 2 + "%)" : 0,
```

Comment:
why do you need calc here?
and maybe you can use some variables for readability